<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GroupchatDialog</class>
 <widget class="QDialog" name="GroupchatDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>491</width>
    <height>529</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>群聊</string>
  </property>
  <widget class="QListWidget" name="listWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>40</y>
     <width>491</width>
     <height>431</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>10</y>
     <width>80</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QToolButton" name="back_Btn">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>51</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>返回</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="inputEdit">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>490</y>
     <width>341</width>
     <height>29</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolButton" name="sendBtn">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>490</y>
     <width>41</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>S</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
